src/main/java/com/company/Main.java

```diff
@@ -0,0 +1,40 @@
+package com.company;
+
+import lombok.extern.slf4j.Slf4j;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
+
+import static com.company.Utility.getNumber;
+
+@Slf4j
+public class Main {
+
+    public static void main(String[] args) throws IOException {
+        BufferedReader reader = new BufferedReader(new InputStreamReader(System.in));
+        String input;
+
+        do {
+            double number1, number2;
+            System.out.println("Please enter the 1st number:");
+            number1 = getNumber(1);
+
+            System.out.println("Choose an operator: +, -, * or /");
+            char operator = reader.readLine().charAt(0);
+            log.info("Operator is " + operator);
+
+            System.out.println("Enter the 2nd number");
+            number2 = getNumber(2);
+
+            Utility.calculate(number1, number2, operator);
+
+            System.out.println("Continue? Y/N");
+            input = reader.readLine();
+        }
+        while (input.equalsIgnoreCase("Y"));
+
+        log.info("Program terminated on user request" + System.lineSeparator());
```

Comment:
new line after each log message must be added in the logback config message format pattern